solver/src/solver/oneinch_solver.rs

```diff
@@ -125,6 +129,11 @@ impl OneInchSolver {
 
         settlement.with_liquidity(&order, order.sell_amount)?;
 
+        let existing_allowance = self
```

Comment:
Is there a reason this moved here? The reason I had it near the beginning was to make the 1Inch API call as late as possible to make price movement less likely. I'm not sure if it has a significant impact however, so maybe it doesn't really matter.

Reply:
I wanted to avoid doing a call to the node if the limit price is not met, happy to move it back up

Reply:
Either way, I don't think it makes much difference. The question becomes: which do we value more?
- Additional node stress when 1Inch price isn't good enough?
- Price older by a single node round trip?

I'm OK with either decision.